### PR TITLE
[WIP] Deepsea integration

### PR DESCRIPTION
**Pull request by dincamihai**
This is a work in progress [Deepsea integration](http://mypy.readthedocs.io/en/latest/cheat_sheet.html) with SUSE Manager

Known issues:

- uses custom qcow2 image (missing from the PR) [fix the kernel problem first] [ref link](https://github.com/moio/sumaform/pull/170/files#diff-daf7f17fcde3f604e71594a880299068R72)
- deepsea terraform module needs to reuse more the minion module [ref link](https://github.com/moio/sumaform/pull/170/files#diff-f402ca4d6fb4cedbfa91ef477ec9ba87)
- the extra volume required by ceph should be defined in the deepsea module

### CODE REVIEW

**Comment by moio** [comment]
@dincamihai please ping me if I can help somehow!

File: modules/libvirt/base/variables.tf [hunk 1]
```diff
@@ -67,8 +67,9 @@ variable "image_locations" {
     sles11sp3 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles11sp3.x86_64.qcow2"
     sles11sp4 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles11sp4.x86_64.qcow2"
     sles12 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12.x86_64.qcow2"
-    sles12sp1 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12sp1.x86_64.qcow2"
+    # sles12sp1 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12sp1.x86_64.qcow2"
     sles12sp2 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12sp2.x86_64.qcow2"
+    sles12sp1 = "/home/store/sles12sp1-2.qcow2"
```
Comment (MalloZup) [inline modules/libvirt/base/variables.tf]: remember to remove this at the end :panda_face:  and use the std

**Comment by moio** [comment]
@dincamihai is there any plan to continue this? Any idea about the timeline?

**Comment by moio** [comment]
@dincamihai I am closing this due to inactivity, feel free to reopen anytime.

Thanks for the efforts so far anyway.

**Pull request closed without merging**